raft/log_test.go

```diff
@@ -453,22 +410,17 @@ func TestCommitTo(t *testing.T) {
 		{4, 0, true},  // commit out of range -> panic
 	}
 	for i, tt := range tests {
-		func() {
-			defer func() {
-				if r := recover(); r != nil {
-					if !tt.wpanic {
-						t.Errorf("%d: panic = %v, want %v", i, true, tt.wpanic)
-					}
-				}
-			}()
-			raftLog := newLog(NewMemoryStorage(), raftLogger)
-			raftLog.append(previousEnts...)
-			raftLog.committed = commit
-			raftLog.commitTo(tt.commit)
-			if raftLog.committed != tt.wcommit {
-				t.Errorf("#%d: committed = %d, want %d", i, raftLog.committed, tt.wcommit)
+		t.Run(fmt.Sprint(i), func(t *testing.T) {
+			if tt.wpanic {
+				newLog(NewMemoryStorage(), raftLogger)
```

Comment:
I don't think it is correct.

The panic happens when tt.commit > 2 [`commit := uint64(2)`]